docs/UserGuide.md

```diff
@@ -226,30 +248,40 @@ Do take some time to read the following note to better understand how you can us
 
 </div>
 
-### 5.1 Client commands
+### 7.1 Client commands
 
 This subsection covers all client-related commands.
 
 #### 5.1.1 Add client: `addClient`
 
-Add a new client to MyInsuRec.
+Add a new client to _MyInsuRec_.
+
+Use case:
+1. Suppose you found a potential client, you can use this command to add their particulars into _MyInsuRec_ so that you would not forget!
 
 Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
 
-* A client must have a `NAME` and a `PHONE_NUMBER`.
+* A client **must** have a `NAME` and a `PHONE_NUMBER`.
 * `EMAIL`, `BIRTHDAY`, `ADDRESS` and `PRODUCT` are optional.
+* If a `Name` already exist in _MyInsuRec_, adding the same `NAME` will result in an error!
```

Comment:
i think we will be updating this to be name phone pair

Reply:
but can confirm it first

Reply:
if phone or email alr exist, cannot add